clients/cobol-lsp-vscode-extension/src/services/copybook/CopybookDownloadService.ts

```diff
@@ -67,6 +82,12 @@ export class CopybookDownloadService {
       return true;
     }
 
+    const res = await this.downloadCopybooksinProcessorGroups(
+      copybookName,
+      documentUri,
+    );
+    if (res) return true;
```

Comment:
If the `downloadCopybooksinProcessorGroups` fails to download a copybook, this test seems to fallback to the Settings based approach even if there is an appropriate processor group configuraiton.